crates/ruff_linter/src/message/mod.rs

```diff
@@ -384,6 +384,76 @@ impl<'a> EmitterContext<'a> {
     }
 }
 
+/// A secondary identifier for a lint diagnostic.
+///
+/// For Ruff rules this means the noqa code.
+#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Default, Hash)]
```

Comment:
```suggestion
#[derive(Clone, Debug, PartialEq, Eq, Default, Hash)]
```

Or do we use the ordering somewhere?

Reply:
I think they're used for adding noqa codes, but I'll double check.

Reply:
Nah, it's fine then.